### ISSUE TITLE
Deep River and Strong Wind explosions

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#1240]
Both the Deep River and Strong Wind appear to be using an explosion type that does not exist.
```
ship "Strong Wind"
	explode "small explosion" 60
	explode "medium explosion" 40
	**explode "large explosion" 20**
```
```
ship "Deep River"
	explode "small explosion" 80
	explode "medium explosion" 60
	**explode "large explosion" 30**
	explode "huge explosion" 20
```

"Large" is not a valid explosion type, and does not cause any explosion effects upon the destruction of a ship. I believe that it is suppose to be "big."

**Comment by Amazinite** [comment]
Just searched the entire repository. These are the only two instances of this.

**Comment by Amazinite** [comment]
Possibly also change the `large final explosion` to `big final explosion` for the sake of consistency.

**Comment by endless-sky** [comment]
In nearly all the things that have five different sizes, I use "tiny, small, medium, large, huge" which has the advantage that the files (e.g. images) are sorted by size if sorted alphabetically. The explosions are one of the few places that pattern isn't followed, although at some point I'll probably fix that.

**Comment by endless-sky** [comment]
Fixed in ef6b850.
